Deployment/ASP.Net_Core_to_EC2/Readme.md

```diff
@@ -224,35 +445,35 @@ the Marketplace.
 
 1.  Configure the Code Deploy task.
```

Comment:
It would be super helpful to explain what the user is configuring for each field (brief sentences, maybe as bullet points)